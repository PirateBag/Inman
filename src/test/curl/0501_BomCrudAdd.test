Add W-003 to the children of W-001.

=== JSON ===
{ "updatedRows" : [
    { "parentId" :  "1", "childId" :  "3",  "quantityPer" :  "1.00", "activityState" :  "INSERT" }
  ] }
