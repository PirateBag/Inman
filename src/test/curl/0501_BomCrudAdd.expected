{
  "responseType" : "CHANGE",
  "errors" : [ ],
  "data" : [ {
    "id" : 4,
    "activityState" : "INSERT",
    "parentId" : 1,
    "childId" : 3,
    "quantityPer" : 1.0,
    "parentSummary" : "W-001",
    "parentDescription" : "36 In Red Wagon",
    "childSummary" : "W-003",
    "childDescription" : "Front Wheel Asmbly",
    "unitCost" : 0.0,
    "extendedCost" : 0.0
  } ]
}